docs/build.md

```diff
@@ -30,6 +30,65 @@ Selecting this button shows a quick pick list for all the targets CMake Tools is
 
 You can build a single target without changing the current build target from the VS Code command pallette by running the **CMake: Build a target** command, or by pressing the keyboard shortcut (default is `Shift+F7`). CMake will build any dependent targets, even if they aren't directly selected.
 
+## Create a build task
+You can also define a build task from the VS Code command pallette by running the **Tasks: Configure task** command.
+
+![Configure a build task](images/configure_task.png)
+
+By choosing the "CMake: builds" task, the following task will be created in "./.vscode/tasks.json" file.
+
+`{
```

Comment:
multiline code blocks should be started with 3 backticks and a newline so that the formatting is preserved.